Source/SchemaAndCodeGenerator.cs

```diff
@@ -72,7 +72,7 @@ public IEnumerable<ExplorerItem> GetItemsAndCode(string nameSpace, string typeNa
 				var excludeCatalogs = (string?)_cxInfo.DriverData.Element(CX.ExcludeCatalogs);
 				if (excludeCatalogs != null) options.ExcludedCatalogs = excludeCatalogs.Split(',', ';');
 
-				options.GetProcedures  = (string?)_cxInfo.DriverData.Element(CX.ExcludeRoutines) != "true";
+				options.GetProcedures  = (string?)_cxInfo.DriverData.Element(CX.ExcludeRoutines) == "false";
 				options.GetForeignKeys = (string?)_cxInfo.DriverData.Element(CX.ExcludeFKs)      != "true";

```

Comment:
Should update this to match the other(s)?

Reply:
nope, only for procedures